bench/CMakeLists.txt

```diff
@@ -11,63 +11,58 @@ include(FindOpenCL)
 include(FindComputeCpp)
 include(ConfigureSYCLBLAS)
 
-# Manually search for CLBLAST
-# if (DEFINED CLBLAST_ROOT) 
-#   set(CMAKE_PREFIX_PATH ${CLBLAST_ROOT})
-# else() 
-#   set(CMAKE_PREFIX_PATH "/usr/lib/clblast")
-# endif()
-
-# pkg_search_module(CLBLAST clblast)
-# pkg_check_modules(CLBLAST clblast)
-
-# if (CLBLAST_FOUND)
-#   add_executable(clblast_benchmarks clblast_benchmark.cpp)
-#   set_property(TARGET clblast_benchmarks PROPERTY CXX_STANDARD 11)
-#   include_directories(${CLBLAST_INCLUDE_DIRS})
-#   target_link_libraries(clblast_benchmarks PUBLIC ${CLBLAST_LIBRARIES})
-#   target_link_libraries(clblast_benchmarks PUBLIC ${OpenCL_LIBRARIES})
-# endif()
-
-# # Manually search for CLBLAS_ROOT
-# if (DEFINED CLBLAS_ROOT) 
-#   set(CMAKE_PREFIX_PATH ${CLBLAS_ROOT})
-# else ()
-#   set(CMAKE_PREFIX_PATH "/usr/lib/clblas")
-# endif()
-
-# pkg_search_module(CLBLAS clBLAS)
-# pkg_check_modules(CLBLAS clBLAS)
+# Add syclblas benchmarks.
+file(GLOB SYCL_BENCHMARK_SRCS
+  ${CMAKE_CURRENT_SOURCE_DIR}/syclblas/syclblas_benchmark_level1.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/syclblas/syclblas_benchmark_level2.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/syclblas/syclblas_benchmark_level3.cpp
+)
+foreach(syclblas_benchmark ${SYCL_BENCHMARK_SRCS})
+  get_filename_component(sycl_bench_exec ${syclblas_benchmark} NAME_WE)
+  set(sycl_bench_exec, ${syclblas_benchmark})
+  add_executable(${sycl_bench_exec} ${syclblas_benchmark})
+  set_property(TARGET ${sycl_bench_exec} PROPERTY CXX_STANDARD 11)
+  add_sycl_to_target(
+    TARGET ${sycl_bench_exec}
+    SOURCES ${syclblas_benchmark}
+  )
+endforeach()
+include_directories(${SYCLBLAS_INCLUDE} ${ComputeCpp_INCLUDE_DIRS})
 
-# if (CLBLAS_FOUND)
-#   pkg_search_module(CLBLAS OPTIONAL clBLAS)
-#   pkg_check_modules(CLBLAS clBLAS OPTIONAL)
-#   add_executable(clblas_benchmarks clblas_benchmark.cpp)
-#   set_property(TARGET clblas_benchmarks PROPERTY CXX_STANDARD 11)
-#   include_directories(${CLBLAS_INCLUDE_DIRS})
-#   target_link_libraries(clblas_benchmarks ${CLBLAS_LIBRARIES})
-#   target_link_libraries(clblas_benchmarks ${OpenCL_LIBRARIES})
-# endif(CLBLAS_FOUND)
+# Try and also build clBLAST benchmarks.
+# Manually search for CLBLAST
+if (DEFINED CLBLAST_ROOT) 
+  message(STATUS "CLBlast root specified as : ${CLBLAST_ROOT}")
+  set(CMAKE_PREFIX_PATH ${CLBLAST_ROOT})
+else() 
+  # set a default path if not specified
+  set(CMAKE_PREFIX_PATH "/usr/lib/clblast")
+endif()
```

Comment:
This else should show a message explaining what path is using by default

Reply:
Done